helix-lock/src/test/java/org/apache/helix/lock/helix/TestZKHelixNonblockingLock.java

```diff
@@ -76,6 +76,7 @@ public void beforeMethod() {
 
   @AfterSuite
   public void afterSuite() throws IOException {
+    System.out.println("afterSuite " + getClass().getName());
```

Comment:
Same here, what's the usage of this output?

Reply:
removed.